### ISSUE TITLE
tree breadcrumb support

### ISSUE THREAD

**Issue by dbu** [sonata-project/SonataDoctrinePhpcrAdminBundle#189]
as discussed in https://github.com/symfony-cmf/MenuBundle/pull/135 we could show the tree elements in the breadcrumb. the perfect thing would be
- [ ] Have a content breadcrumb (content ancestor) in parallel with the administrative breadcrumb
- [ ] check if allowed to edit / view and if neither display entry without link
- [ ] configure number of entries in the breadcrumb, if there is more than that, remove some in the middle and put a non linked "..." entry there
- [ ] show tree with location of this document (on the left side?, instead of the parent selector?)

**Comment by ElectricMaxxx** [comment]
Do we have a breadcrum view at all?

**Comment by dbu** [comment]
![breadcrumb](https://cloud.githubusercontent.com/assets/76576/22596824/681330b8-ea2d-11e6-9eeb-6b762e097d1d.png)

this. its following the code structure, not the content hierarchy.

**Comment by dbu** [comment]
but this is really a nice to have and not critical for 2.0 at all. i remove the milestone

**Comment by ElectricMaxxx** [comment]
For looking for ancestors, parents, i could eventually reuse the code
from the suggestion provider in seo-bundle's sitemap handling?

**Comment by ElectricMaxxx** [comment]
Maybe you would just add a `NiceToHave` label and keep the milestone
instead?

**Comment by dbu** [comment]
dunno. we can do it for 2.1 too, no point in cluttering up the milestone.
rather release sooner.

Maybe you would just add a `NiceToHave` label and keep the milestone
instead?

**Comment by ElectricMaxxx** [comment]
i created a milestone for that "new" stuff so we won't loose that.

**Comment by dbu** [comment]
would be nice but not particularly important
